---
version: 1
interactions:
- request:
    body: <CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><LocationConstraint>fr-par</LocationConstraint></CreateBucketConfiguration>
    form: {}
    headers:
      Content-Length:
      - "150"
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Bucket-Object-Lock-Enabled:
      - "false"
      X-Amz-Content-Sha256:
      - 2cb57fad7b7168921a4c94426cfcb9ee2953f126430595df844e22d50f029060
      X-Amz-Date:
      - 20221110T142307Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 10 Nov 2022 14:23:10 GMT
      Location:
      - /test-acc-scaleway-object-cors-empty-origin-1223290516769971322
      X-Amz-Id-2:
      - tx1183a779cbb1452bb7afb-00636d094b
      X-Amz-Request-Id:
      - tx1183a779cbb1452bb7afb-00636d094b
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      Content-Md5:
      - 1B2M2Y8AsgTpgAmY7PhCfg==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20221110T142310Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?acl=
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 10 Nov 2022 14:23:12 GMT
      X-Amz-Id-2:
      - tx404e5f332bc74f2a82d1d-00636d094e
      X-Amz-Request-Id:
      - tx404e5f332bc74f2a82d1d-00636d094e
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142312Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txd478f8c08db8416fa98f6-00636d0950</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:12 GMT
      X-Amz-Id-2:
      - txd478f8c08db8416fa98f6-00636d0950
      X-Amz-Request-Id:
      - txd478f8c08db8416fa98f6-00636d0950
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142312Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx0cd9878ac9574d9c823be-00636d0952</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:14 GMT
      X-Amz-Id-2:
      - tx0cd9878ac9574d9c823be-00636d0952
      X-Amz-Request-Id:
      - tx0cd9878ac9574d9c823be-00636d0952
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142312Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx70d72b2c83494a7ea923f-00636d0956</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:18 GMT
      X-Amz-Id-2:
      - tx70d72b2c83494a7ea923f-00636d0956
      X-Amz-Request-Id:
      - tx70d72b2c83494a7ea923f-00636d0956
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142312Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txc6c9672b9dc348958df6b-00636d095e</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:27 GMT
      X-Amz-Id-2:
      - txc6c9672b9dc348958df6b-00636d095e
      X-Amz-Request-Id:
      - txc6c9672b9dc348958df6b-00636d095e
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142327Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx2a3c7ba2495e48f6bbc51-00636d095f</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:27 GMT
      X-Amz-Id-2:
      - tx2a3c7ba2495e48f6bbc51-00636d095f
      X-Amz-Request-Id:
      - tx2a3c7ba2495e48f6bbc51-00636d095f
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142327Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txba2fc18a47e9440a9bde9-00636d0961</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:29 GMT
      X-Amz-Id-2:
      - txba2fc18a47e9440a9bde9-00636d0961
      X-Amz-Request-Id:
      - txba2fc18a47e9440a9bde9-00636d0961
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142327Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx64696f084bf94a6a9437e-00636d0965</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:33 GMT
      X-Amz-Id-2:
      - tx64696f084bf94a6a9437e-00636d0965
      X-Amz-Request-Id:
      - tx64696f084bf94a6a9437e-00636d0965
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142327Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txe44f07778eb0447894785-00636d096d</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:42 GMT
      X-Amz-Id-2:
      - txe44f07778eb0447894785-00636d096d
      X-Amz-Request-Id:
      - txe44f07778eb0447894785-00636d096d
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142342Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx342dd0eaac334f7fa91af-00636d096e</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:42 GMT
      X-Amz-Id-2:
      - tx342dd0eaac334f7fa91af-00636d096e
      X-Amz-Request-Id:
      - tx342dd0eaac334f7fa91af-00636d096e
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142342Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txd3f199f6c5034ee9a98d6-00636d0970</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:45 GMT
      X-Amz-Id-2:
      - txd3f199f6c5034ee9a98d6-00636d0970
      X-Amz-Request-Id:
      - txd3f199f6c5034ee9a98d6-00636d0970
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142342Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx3afc28904ae4492693734-00636d0975</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:23:59 GMT
      X-Amz-Id-2:
      - tx3afc28904ae4492693734-00636d0975
      X-Amz-Request-Id:
      - tx3afc28904ae4492693734-00636d0975
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142342Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txf11630a768754b4089c27-00636d0987</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:24:07 GMT
      X-Amz-Id-2:
      - txf11630a768754b4089c27-00636d0987
      X-Amz-Request-Id:
      - txf11630a768754b4089c27-00636d0987
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142407Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txccadc504a95847fba68c3-00636d0987</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:24:07 GMT
      X-Amz-Id-2:
      - txccadc504a95847fba68c3-00636d0987
      X-Amz-Request-Id:
      - txccadc504a95847fba68c3-00636d0987
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142407Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx3025b829a1ec4016a9500-00636d0989</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:24:09 GMT
      X-Amz-Id-2:
      - tx3025b829a1ec4016a9500-00636d0989
      X-Amz-Request-Id:
      - tx3025b829a1ec4016a9500-00636d0989
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142407Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txfa12eda348f040178a703-00636d098d</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:24:14 GMT
      X-Amz-Id-2:
      - txfa12eda348f040178a703-00636d098d
      X-Amz-Request-Id:
      - txfa12eda348f040178a703-00636d098d
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20221110T142407Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txc51c49bd5fc245f8914e8-00636d0996</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 10 Nov 2022 14:24:22 GMT
      X-Amz-Id-2:
      - txc51c49bd5fc245f8914e8-00636d0996
      X-Amz-Request-Id:
      - txc51c49bd5fc245f8914e8-00636d0996
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      User-Agent:
      - aws-sdk-go/1.44.128 (go1.19.2; darwin; amd64)
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20221110T142422Z
    url: https://test-acc-scaleway-object-cors-empty-origin-1223290516769971322.s3.fr-par.scw.cloud/
    method: DELETE
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 10 Nov 2022 14:24:27 GMT
      X-Amz-Id-2:
      - tx100282eca06e4c65ba50f-00636d0996
      X-Amz-Request-Id:
      - tx100282eca06e4c65ba50f-00636d0996
    status: 204 No Content
    code: 204
    duration: ""
